examples/image_classifier/streaming_video_client_side_batching/README.md

```diff
@@ -0,0 +1,80 @@
+# Streaming Video Inference with client-side batching
+
+To use TorchServe's dynamic batching feature, please refer to [resnet_152_batch](https://github.com/pytorch/serve/tree/examples/streaming_video/examples/image_classifier/resnet_152_batch)
+
+For some near real-time streaming video applications, there might be a need for batching on the application side.
+This example shows how this can be achieved using TorchServe.
+
+On the client side, we have one thread for reading frames from a video source and another thread which batches(size n) the read frames and sends the request to TorchServe for image classification inference.
+To send the batched data, we create a json payload of n frames.
+On the TorchServe side, we read the json payload and preprocess the n frames. The postprocess function in the handler returns the output as a list of length 1.
+
+### Client application using [OpenCV](https://opencv.org/)
+
+In this example, we are using OpenCV to send frames on the client side.
+Install OpenCV with the following command
+```
+pip install opencv-python
+```
+
+### Create a resnet-18 eager mode model archive, register it on TorchServe and run inference on a streaming video
+
+Run the commands given in following steps from the parent directory of the root of the repository. For example, if you cloned the repository into /home/my_path/serve, run the steps from /home/my_path
+
+```bash
+wget https://download.pytorch.org/models/resnet18-f37072fd.pth
```

Comment:
Might just make senses to have a setup script bash script which runs everything until torchserve --start and then in a seperate window users can run `request.py`

Reply:
Done.